pkg/model/devrc.go

```diff
@@ -0,0 +1,246 @@
+package model
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"strings"
+	"time"
+
+	"github.com/okteto/okteto/pkg/log"
+	yaml "gopkg.in/yaml.v2"
+	apiv1 "k8s.io/api/core/v1"
+)
+
+// Dev represents a development container
+type DevRC struct {
+	Autocreate           *bool                 `json:"autocreate,omitempty" yaml:"autocreate,omitempty"`
```

Comment:
Let's just support labels, annotations, context, namespace, environment, secrets, syncs, resources, PersistentVolumeInfo and timeout for now